extensions/todoist/src/components/TaskDetail.tsx

```diff
@@ -72,6 +73,14 @@ export default function TaskDetail({ taskId }: TaskDetailProps): JSX.Element {
 
                 <Detail.Metadata.Label title="Due Date" text={displayedDate} icon={Icon.Calendar} />
 
+                {subTasks && subTasks?.length > 0 ? (
+                  <Detail.Metadata.Label
+                    title={"Sub-tasks"}
+                    text={`${subTasks.length} subtask` + (subTasks.length > 1 ? "s" : "")}
```

Comment:
What do you think about: ``${subTasks.length} subtask${subTasks.length > 1 ? "s" : ""}``

Reply:
im pretty new to JS in general so never sure whats best practice. lt me modify and push